app/package.json

```diff
@@ -12,6 +12,7 @@
   "dependencies": {
     "axios": "~0.21.1",
     "bodybuilder": "^2.4.0",
+    "i": "^0.3.6",
```

Comment:
What is the i dependency?

Reply:
This was not intended. I will remove this.